types/human-date/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for human-date 1.4
+// Project: https://github.com/montanaflynn/human-date
+// Definitions by: Rico Sandyca Novenza <https://github.com/ricosandyca>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface PrettyPrintOptions {
+    showTime: boolean;
+}
+
+export interface RelativeTimeOptions {
+    futureSuffix: string;
+    pastSuffix: string;
+    presentText: string;
+    returnObject: boolean;
+    allUnits: boolean;
+}
```

Comment:
```suggestion
export interface RelativeTimeOptions {
    futureSuffix?: string;
    pastSuffix?: string;
    presentText?: string;
    returnObject?: boolean;
    allUnits?: boolean;
}
```
as per implementation sample, these are optional:
```ts
hdate.relativeTime(4, {futureSuffix: "in the future"});
hdate.relativeTime(new Date("8-16-1987"), {returnObject: true});
```
https://github.com/montanaflynn/human-date/blob/master/humandate.js